openshift/templates/matchmaking/api/mm-api-dc-temaple.yaml

```diff
@@ -1 +1,97 @@
-## Draft
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: ${APPLICATION_NAME}-template
+labels:
+  template: ${APPLICATION_NAME}-template
+objects:
+
+- kind: DeploymentConfig
+  apiVersion: apps.openshift.io/v1
+  metadata:
+    annotations:
+      app.openshift.io/route-disabled: 'false'
+      openshift.io/generated-by: OpenShiftWebConsole
+    name: ${APPLICATION_NAME}
+    generation: 2
+    namespace: ${LICENSE_PLATE}-${ENVIRONMENT}
+    labels:
+      app: ${APPLICATION_NAME}
+      app.kubernetes.io/part-of: match-making
+  spec:
+    strategy:
+      type: Rolling
+      rollingParams:
+        updatePeriodSeconds: 1
+        intervalSeconds: 1
+        timeoutSeconds: 600
+        maxUnavailable: 25%
+        maxSurge: 25%
+      resources: {}
+      activeDeadlineSeconds: 21600
+    triggers:
+      - type: ImageChange
+        imageChangeParams:
+          automatic: true
+          containerNames:
+            - ${APPLICATION_NAME}
+          from:
+            kind: ImageStreamTag
+            namespace: ${LICENSE_PLATE}-tools
+            name: '${IMAGESTREAM}:latest'
+          lastTriggeredImage: >-
+            image-registry.openshift-image-registry.svc:5000/ec1236-tools/match-making-api@sha256:c3d57523a0277faacef24e3703b31f42dbfb3fbde88b3c27036886f7abaad78c
+      - type: ConfigChange
+    replicas: 1
+    revisionHistoryLimit: 10
+    test: false
+    selector:
+      app: ${APPLICATION_NAME}
+      deploymentconfig: ${APPLICATION_NAME}
+    template:
+      metadata:
+        creationTimestamp: null
+        labels:
+          app: ${APPLICATION_NAME}
+          deploymentconfig: ${APPLICATION_NAME}
+        annotations:
+          openshift.io/generated-by: OpenShiftWebConsole
+      spec:
+        containers:
+          - name: ${APPLICATION_NAME}
+            image: >-
+              image-registry.openshift-image-registry.svc:5000/ec1236-tools/match-making-api@sha256:c3d57523a0277faacef24e3703b31f42dbfb3fbde88b3c27036886f7abaad78c
+            ports:
+              - containerPort: 8080
```

Comment:
The matchmaking api container has a port of 8080?

Reply:
default value of the container port